components/brave_wallet/browser/brave_wallet_provider_impl.h

```diff
@@ -47,7 +45,10 @@ class BraveWalletProviderImpl final : public mojom::BraveWalletProvider,
  private:
   std::unique_ptr<BraveWalletProviderDelegate> delegate_;
   mojo::Remote<mojom::EventsListener> events_listener_;
-  EthJsonRpcController* rpc_controller_;  // NOT OWNED
+  mojo::Remote<mojom::EthJsonRpcController> rpc_controller_;
+  //
+  // Receiver for the CrosNetworkConfigObserver events.
```

Comment:
Cros network config observer?

Reply:
Fixed in 9db58249b3823223ece2b39fda6b68bf9f5d3ee0